test_cases/encoding.json

```diff
@@ -45,10 +45,13 @@
     },
     {
       "id": 3,
-      "status": "pass",
+      "status": "fail",
       "endpoint": "search",
       "issue": "https://github.com/pelias/api/issues/600",
```

Comment:
this issue has been closed once the test started passing. please add the new issue pelias/pelias#426 to the list. just change `issue` to be an array of links.